src/api/example/controllers/json-controller.js

```diff
@@ -0,0 +1,30 @@
+import Joi from 'joi'
+
+export const getJsonController = {
+  handler: (request, h) => {
+    return h
+      .response({
+        key: '123',
+        value: 'abc',
+        bool: true,
+        number: 42,
+        array: [1, 2, 3],
+        obj: { foo: 'bar' }
+      })
+      .code(200)
+  }
+}
+
+export const sendJsonController = {
```

Comment:
nit: worth naming this `postJsonController`?

Reply:
i avoid post since its endpoint also accepts put and patch